### PR TITLE
Solved issue 1

### LINKED ISSUE #1: Crawler Update
New data to be collected
- [x] Player Id
- [x] Player Short names
- [x] Most played player position
- [x] Club and country logo URLs

Old Data to be fixed
 - [x] TBD

### PR DESCRIPTION
**Pull request by Akram9**
Hi Saurav,

I have solved issue 1 in this PR.

### CODE REVIEW

File: fifa-crawler/fifa_parser/spiders/players_stats.py [hunk 1]
```diff
@@ -52,6 +55,26 @@ def parse(self, response):
...
+            for k, v in zip(player_clubs_list, player_logos_list):
+                if k == player_country:
+                    nationalClub = {'name': k, 'url': v}
+                    player_logos_urls.update({'nationalClub': nationalClub})
+                else:
+                    club = {'name': k, 'url': v}
+                    player_logos_urls.update({'club': club})
+
+            # Add player most played position
+            player_best_position = player.xpath('//div[@class="columns right"]/div[@class="column col-4"]//div[@class="double-spacing"]//span/text()').get()
```
Comment (sauravhiremath) [inline fifa-crawler/fifa_parser/spiders/players_stats.py]: Can u do the renaming here too?

```suggestion
            player_primary_position = player.xpath('//div[@class="columns right"]/div[@class="column col-4"]//div[@class="double-spacing"]//span/text()').get()
```

And subsequently changing line136 too

**Review by sauravhiremath — CHANGES_REQUESTED** [review]
LGTM

**Comment by Akram9** [comment]
I have renamed the variable as requested.

**Comment by sauravhiremath** [comment]
Nice work @Akram9 :rocket: 

Fixes #1

**Pull request merged**